cmake/BareosCpmPackages.cmake

```diff
@@ -39,3 +39,20 @@ CPMAddPackage(
   GIT_TAG 10.2.1
   EXCLUDE_FROM_ALL YES
 )
+
+CPMAddPackage(
+  NAME CLI11
+  VERSION 2.4.2
+  GITHUB_REPOSITORY CLIUtils/CLI11
+  GIT_TAG v2.4.2
```

Comment:
`GIT_TAG` defaults to `v$VERSION`, so if that is the same you do not need to explicitly set it.

Reply:
thanks fixed